src/runtime/server/middleware/cspNonceHandler.ts

```diff
@@ -5,7 +5,6 @@ import { getRouteRules } from '#imports'
 
 export type NonceOptions = {
   enabled: boolean;
-  mode: 'renew' | 'check';
   value: undefined | (() => string);
```

Comment:
I'm not sure about giving the developper a way to override crypto primitives which are vetted to be safe

Reply:
IMO that's out of scope of this PR